internal/internal.go

```diff
@@ -429,7 +429,16 @@ func (p *envoyExtAuthzGrpcServer) log(ctx context.Context, input interface{}, re
 		info.Metrics = result.metrics
 	}
 
-	if err == nil {
```

Comment:
Nit: the `if result != nil` check above can be removed since the `Check` function always passes a result to the `log` function now.

Reply:
Removed